multinode-demo/validator.sh

```diff
@@ -16,6 +16,13 @@ if [[ "$1" = "-h" || -n "$3" ]]; then
   usage
 fi
 
+if [[ "$1" == "-x" ]]; then
```

Comment:
double quotes around `$1` not needed right?

Reply:
Let's mention `-x` in `usage` for some self-documentation please

Reply:
I was adhering to the style of the rest of the script, which is full of double quoting inside [[